### ISSUE TITLE
LEMFO LEM 12 support

### ISSUE THREAD

**Issue by VolodKu** [OpenWatchProject/manifest#2]
Great Project.

I saw android_device_wiite_c7s is compatible with LEM 12 watch.
Then my LEM 12 watch is based on arm-v7 and Android 7.1.
But after building, the image is based on Android 10.0 and arm64.
I am very curious of this.
The full_c7s supports the LEM12 watch?

Thanks.

**Comment by iscle** [comment]
Yes, C7S builds support the LEM12, and yes, they are arm64 and Android 10 :)

BTW, sorry for the late reply, I've just seen the issue!

**Comment by VolodKu** [comment]
But my LEM12 has armv7 and Android 7 device.
Also partition table is different.
Can those work?

**Comment by iscle** [comment]
Yes, all LEM12 are like that from the factory.

You will need to flash it by using the Format All + Download option, but write down your imei before doing so!

**Comment by VolodKu** [comment]
But while flashing, I am getting error blocking.

**Comment by iscle** [comment]
How are you flashing it?

**Comment by VolodKu** [comment]
format all and download

**Comment by VolodKu** [comment]
Do you have some built firmware files?

**Comment by VolodKu** [comment]
So I did try to flash by MTK Flash tool. The tools is requiring the format since Partition Table changed. While flashing, I am getting errors. So If you have some firmwares, I can test on my watch.
Thanks.

**Comment by ghost** [comment]
> So I did try to flash by MTK Flash tool. The tools is requiring the format since Partition Table changed. While flashing, I am getting errors. So If you have some firmwares, I can test on my watch. Thanks.

Hey. Could you please send me Android 10 by email? I would really like to test it out.

**Comment by ghost** [comment]
> So I did try to flash by MTK Flash tool. The tools is requiring the format since Partition Table changed. While flashing, I am getting errors. So If you have some firmwares, I can test on my watch. Thanks.

Hey. Could you please send me Android 10 by email? I would really like to test it out.
???
